backend/fms_core/coordinates.py

```diff
@@ -58,6 +58,70 @@ def ints(end: int, pad_to: int = 0) -> CoordinateAxis:
 
     return tuple(str(i).zfill(pad_to) for i in range(1, end + 1))
 
+def is_alpha_digit_spec(spec: CoordinateSpec) -> bool:
+    '''Determines if a CoordinateSpec is for the alpha/digit style, eg "A01, B12, etc..."'''
+    if len(spec) == 2:
+        alphas = spec[0]
+        digits = spec[1]
+
+        all_alphas = ''.join(alphas).isalpha()
+        all_digits = ''.join(digits).isdigit()
+
+        return all_alphas and all_digits
+
+    return False
+
+def convert_alpha_digit_coord_to_ordinal(coord: str, spec: CoordinateSpec) -> int:
+    '''
+    Convert a coordinate with the alpha/digit style (eg. A01) to an integer value, starting at 1.
+    The coordinate spec must support this style of coordinate.
+    The coordinate is expected to be valid.
+    '''
+
+    # Coordinate must be at least two chars long (eg "A1")
+    if len(coord) < 2:
+        raise CoordinateError(f'Cannot convert coord {coord} to ordinal - bad coord format.')
+    
+    # Spec must be for "A01" style coordinates.
+    if not is_alpha_digit_spec(spec):
+        raise CoordinateError(f'Cannot convert coord {coord} to ordinal - CoordinateSpec does not support coordinate style.')
+    spec_letters = spec[0]
+    spec_digits = spec[1]
+    
+    # Collect the characters at the start of the coord.
+    letters = ''
+    digits = ''
+    for index in range(0, len(coord)):
```

Comment:
Yes, it's overly complex, but I wanted to build in support for coordinates like "AB001" in case we need to support flowcells with thousands of coordinates.